packages/api/src/idpuser/controller.test.ts

```diff
@@ -1,15 +1,17 @@
-/** @format */
-
 import request from "supertest";
 import type { NextFunction, Request } from "express";
 import { app } from "../app";
 import { verifyUserAuthentication } from "../middleware";
-import type { TwoFactorRequest, TwoFactorRequestResponse } from "./models";
-import { fusionAuthClient } from "../fusionauth";
+import type { TwoFactorRequestResponse } from "./models";
+import { getTwoFactorMethods } from "./service";
+
+interface TwoFactorRequest {
+  emailFromAuthToken: string;
+}
 
 jest.mock("node-fetch", () => jest.fn());
 jest.mock("../middleware");
-jest.mock("../fusionauth");
+jest.mock("./service");
 
 describe("/idpuser", () => {
```

Comment:
There should be a test in here covering the case where `getTwoFactorMethods` throws an error (in particular, I'm thinking of the case where the FusionAuth call fails unexpectedly).